account_vat_on_payment/__openerp__.py

```diff
@@ -73,6 +55,16 @@
         'test/account_invoice_7.yml',
         'test/account_invoice_8.yml',
         'test/account_invoice_9.yml',
+        'test/account_invoice_1_real.yml',
+        'test/account_invoice_2_real.yml',
+        'test/account_invoice_3_real.yml',
+        'test/account_invoice_4_real.yml',
+        'test/account_invoice_5_real.yml',
+        'test/account_invoice_6_real.yml',
+        'test/account_invoice_7_real.yml',
+        'test/account_invoice_8_real.yml',
+        'test/account_invoice_1_real_same_account.yml',
```

Comment:
Report: just tried with http://pastebin.com/K4JEJq6H and got
```
odoo/addons/account/account.py", line 1356, in button_validate
    _('You cannot validate this journal entry because account "%s" does not belong to chart of accounts "%s".') % (account.name, top_common.name))
except_orm: ('Error!', u'You cannot validate this journal entry because account "Bank Current Account - (test)" does not belong to chart of accounts "shadow COA".')
```